src/server/debug/server/server.go

```diff
@@ -822,7 +832,7 @@ func (s *debugServer) getWorkerPodsLoki(ctx context.Context, pipelineInfo *pps.P
 	// logs for further inspection.  The alternative would be to get every worker that existed
 	// in some time interval, but that results in too much data to inspect.
 
-	queryStr := `{pipelineName="` + pipelineInfo.Pipeline.Name + `"}`
+	queryStr := fmt.Sprintf(`{pipelineProject="%s", pipelineName="%s"}`, pipelineInfo.Pipeline.Project.Name, pipelineInfo.Pipeline.Name)
```

Comment:
So, I know your quoting algorithm is correct based on the constraints on project and pipeline names, but here's Google's style guide's take: https://google.github.io/styleguide/go/decisions#use-q

To me it's a wash on whether you should rewrite this, but it does save you 4 bytes ;)

Reply:
My reasoning here is that this particular example is creating a LogQL string, and while LogQL and Go both assign similar meanings to strings within double quotes, it is a coincidence. If project or pipeline names could have any characters which would be invalid in LogQL, I would pass them to a QuoteLogQL function …

… I should do that anyway, shouldn’t I? That way it’s obvious when one reads it, and it is correct in the future. Although it’ll cost a lot more than four bytes of code☺

Reply:
I ended up doing exactly that: adding `quoteLogQLStreamSelector`, which just prints using Go’s `%q`.